app/infra/httpService/httpService.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { ipcRenderer } from 'electron';
+import uuid4 from 'uuid4';
+import { ipcConsts } from '../../vars';
+
+class HttpService {
+  static sendRequest({ url, params }: { url: string, params: Object | null }) {
```

Comment:
Optional parameter should be something like: 
{ ..., params: ?Object }

Reply:
100%